test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -432,7 +432,7 @@ const RESPONSE_OPENRTB_NATIVE = {
   ]
 };
 
-describe('S2S Adapter', function () {
+describe.skip('S2S Adapter', function () {
```

Comment:
Is the fixing of tests still in progress?

I removed the skip and noticed that the new test `updates s2sConfig bidders by filtering out if already associated with a previous s2sConfig` and it is failing. 

I'll continue to test it using a test page, but seems as though this test _should_ work if the functionality is there.